test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,30 @@
+var should = require('should');
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var stubBlogFeedLoader = require('./stub-blog-feed-loader.js');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var homePage, blogSection;
+
+  beforeEach(function() {
+    homePage = stubContext.render(HomePage);
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
+    it("should display featured post", function() {
+      blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.not.eql("");
+    });
+    it("should display 3 other latest posts", function() {
+      blogSection.getDOMNode().querySelectorAll(".recent-posts .post-title").length.should.equal(3);
+    });
```

Comment:
@toolness I tried the following and got errors

```javascript
describe("blogSection", function() {
  process.tick(function() {
    it("should display featured post", function() {
      blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.not.eql("");
    });
    it("should display 3 other latest posts", function() {
      blogSection.getDOMNode().querySelectorAll(".recent-posts .post-title").length.should.equal(3);
    });
  });
});
```

![screen shot 2015-05-28 at 11 48 43 am](https://cloud.githubusercontent.com/assets/2896608/7869092/66be6248-0535-11e5-963a-b22fa8105b97.png)

I even tried commenting out `stubContext.unmount` code but it didn't help.  whyyyyyyyyy